app/models/purl.rb

```diff
@@ -97,10 +97,10 @@ def false_targets
   end
 
   # add the release tags, and reuse tags if already associated with this PURL
-  def refresh_release_tags(releases)
-    [true, false].each do |type|
-      releases[type.to_s.to_sym].sort.uniq.each do |release|
-        release_tags << ReleaseTag.for(self, release, type)
+  def refresh_release_tags(actions)
+    ['index', 'delete'].each do |type|
```

Comment:
And we want string keys here since the hash is being converted from a JSON string, right?

Reply:
exactly.